docs/shared/recipes/root-level-scripts.md

```diff
@@ -0,0 +1,71 @@
+# Run Root-Level NPM Scripts with Nx
```

Comment:
I don't know how detailed the recipe should be (cause it's a recipe) but there are several things we should cover.

- How to enable it (you talk about it, but you don't mention that it just defines a project at the root, which I think might be helpful. Not sure about it)
- How to configure inputs and outputs, so I"d use a target that depends on some inputs and produces some outputs.
- How to wire up a root target in the task pipeline. For instance, the root project can depend on other projects. If you have a target called, say, docs, it can depends on all projects and its outputs can be the docs folder.

Turbo docs cover this use. Maybe we can see how they cover it?

Reply:
doesn't the root project depend on every project inside of it by default?